client/state/test/initial-state.js

```diff
@@ -553,7 +561,7 @@ describe( 'initial-state', () => {
 
 			clock.tick( SERIALIZE_THROTTLE );
 
-			expect( setStoredItemSpy ).toHaveBeenCalledTimes( 1 );
+			expect( clearStorageSpy ).toHaveBeenCalledTimes( 1 );
```

Comment:
Shouldn't this test still test if `setStoredItem` was called? We now clear the storage first and _then_ store the state.

Reply:
yup, missed that. working on it now.